toolchain/semantics/testdata/basics/verbose.carbon

```diff
@@ -2,7 +2,7 @@
 // Exceptions. See /LICENSE for license information.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
-// ARGS: -v dump raw-semantics-ir %s
+// ARGS: -v compile --phase=check --dump-raw-semantics-ir %s
```

Comment:
```suggestion
// ARGS: -v compile --phase=check %s
```

This is only looking at `-v` output, the `--dump` shouldn't be needed given `--phase`.